calculator-service/src/main/java/example/calculator/Calculator.java

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
This is a generated file, it would be probably better not to put it to the repository. If it is easy, consider to put generated sources into the `target` directory.

Reply:
I've chosen a separate src directory because the manual code is dependent on the generated sources. Therefore a directory outside target is better in my opinion.